l2-frontend/src/pages/ManageAnalyzers/components/Row.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <tr>
+    <td>
+      {{ data.label }}
+    </td>
+    <td>
+      <ul style="padding-left: 0; list-style-type: none;">
+        <li
+          v-for="g in status"
+          :key="g.id"
+        >
+          {{ g }}
+        </li>
+      </ul>
+    </td>
+    <td>
+      <div class="main-data">
+        <button
+          v-tippy
+          class="btn btn-blue-nb"
+          :value="data"
+          @click="restartAnalyze(data)"
+        >
+          Перезагрузить
+        </button>
+        <button
+          v-tippy
+          class="btn btn-blue-nb"
+          :value="data"
+          style="position: absolute;"
+          @click="getStatus(data)"
+        >
+          Cтатус
```

Comment:
Скорее как то так
```suggestion
          Обновить статус
```
Или иконку добавить